connectors/file/translator-excel/src/main/java/org/teiid/translator/excel/ExcelMetadataProcessor.java

```diff
@@ -57,12 +57,16 @@
     @ExtensionMetadataProperty(applicable=Column.class, datatype=Integer.class, display="Cell Number", description="Cell number, where the column information is defined. If column name is ROW_ID, define it as -1", required=true)    
 	public static final String CELL_NUMBER = MetadataFactory.EXCEL_URI+"CELL_NUMBER"; //$NON-NLS-1$
 
+    @ExtensionMetadataProperty(applicable=Column.class, datatype=Boolean.class, display="Allow Empty Column Name", description="Allow the column information to contain empty cells.")
```

Comment:
This is not required, this is only needed if you are adding additional metadata proeprties